setup-disk.in

```diff
@@ -2,6 +2,7 @@
 
 PREFIX=
 . "$PREFIX/lib/libalpine.sh"
+. "$PREFIX/sbin/setup-dasd"
```

Comment:
I don't think we should implement this as a `/sbin/setup-dasd`. Maybe better to have it as `/lib/dasd-functions.sh` or similar.

Reply:
Agreed.